docs/devGuide/projectManagement.md

```diff
@@ -99,17 +99,29 @@ For general best practices, refer to the guide [_Working with PRs_ @SE-EDU](http
 
 ## Doing a Release
 
-<box type="important" seamless>
+1. **Make sure you have the correct permissions** for [MarkBind's repository](https://github.com/MarkBind/markbind) and [npm](https://www.npmjs.com/)
+    * For Markbind, you need rights to **push to master branch** and **make new releases**
+    <box type="tip" seamless>
 
-**Attention new maintainers!** Ensure that:
+    To check your github permissions, follow the instructions [here](https://docs.github.com/en/rest/collaborators/collaborators?apiVersion=2022-11-28#get-repository-permissions-for-a-user).
```

Comment:
Any simpler way to check? Or could we provide the curl command/link that dev can visit by replacing the param with their username to get the result? Linking the API docs here is nice but would be better if we give a simple copy-paste validation for the future maintainer, wdyt?

Reply:
There is a simpler way to do this according to [stackoverflow](https://stackoverflow.com/a/40326507) which is just to check if you can edit a file on the repo. I was a bit reluctant to write this here because I'm not sure how legitimate this way is. 
But this does seem a lot easier to just do a quick check like this, rather than have to do a curl.

And I was concerned that the curl command/link would change due to API changes. 

TBH I'm kind of leaning towards using the 'just check if you can edit a file way' because its a lot easier, rather than having to like do a curl, get git hub token and so on.

What do you all think!

Reply:
I suppose your point on making a release might suffice (not sure)? if so it's a easy way to verify by checking whether the maintainer can make release -> probably have the same rights to push to master

Reply:
Perhaps using a list here could make it slightly more readable in my opinion e.g.
To check:
- Github permissions...
- If you can make a new release....

Reply:
> I suppose your point on making a release might suffice (not sure)? if so it's a easy way to verify by checking whether the maintainer can make release -> probably have the same rights to push to master

Hm I think you are right @tlylt according to this [documentation](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository)

I'll just remove this check that since if you can do a release - you should be able to push to master.